.github/workflows/pr-auto-assign.yml

```
@@ -0,0 +1,13 @@
+name: 'Auto Assign PR to author'
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-assignee:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
```

IMO this kind of action should be quick or fail fast

```suggestion
    timeout-minutes: 1
```